testkit/src/main/scala/org/bitcoins/testkit/util/BitcoindRpcTest.scala

```diff
@@ -1,22 +1,25 @@
 package org.bitcoins.testkit.util
 
-import akka.actor.ActorSystem
 import org.bitcoins.core.config.NetworkParameters
 import org.bitcoins.rpc.client.common.BitcoindRpcClient
 import org.bitcoins.testkit.rpc.BitcoindRpcTestUtil
 import org.scalatest.{AsyncFlatSpec, BeforeAndAfterAll}
 import org.slf4j.{Logger, LoggerFactory}
 
 import scala.collection.mutable
-import scala.concurrent.duration.DurationInt
-import scala.concurrent.{Await, ExecutionContext}
+import scala.concurrent.Await
+import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext
 
 abstract class BitcoindRpcTest extends AsyncFlatSpec with BeforeAndAfterAll {
   protected val logger: Logger = LoggerFactory.getLogger(getClass)
 
-  implicit val system: ActorSystem =
-    ActorSystem(getClass.getSimpleName, BitcoindRpcTestUtil.AKKA_CONFIG)
-  implicit val ec: ExecutionContext = system.dispatcher
+  /** AsyncFlatSpec provides its own execution context, but for some
+    * reason that doesn't allow doing anything in `afterAll`. We add
+    * our own.
+    */
+  implicit val ec: ExecutionContext = ExecutionContext.Implicits.global
```

Comment:
It worries me to see so many things using this `ExecutionContext` now that we don't have `ActorSystem`s because if this gets backlogged really bad things happen (generally deadlock is inevitable in these cases)